shared/templates/template_OVAL_accounts_password

```diff
@@ -1,5 +1,11 @@
+{{% if rule_id -%}}
+{{% set oval_id = rule_id -%}}
+{{% else -%}}
+{{% set oval_id = "accounts_password_pam_" + VARIABLE -%}}
+{{% endif -%}}
```

Comment:
I assume this construction is here due to so-called unowned templates (I mean shared/templates/extra_ovals.yml). 
At this moment it isn't used there. I know we need to be ready for the future. However, I don't want it here. I would prefer a generic solution instead.

The top level keys in shared/templates/extra_ovals.yml are fake rule IDs in fact. They would be rule IDs if the generated OVAL definition would be owned by a rule. So I think that the ssg/templates.py script can use the top level keys and substitute them to rule_id or some different variable so that it can substitute it into the OVAL template as the OVAL definition ID.

Reply:
I see, this is a good point. Technically I would prefer to use `oval_id`, as `rule_id` creates an expectation that it is indeed an ID of a rule.
I agree that it is more appropriate to put the OVAL ID definition to `templates.py`.

Reply:
I can confirm that `extra_ovals` act as an input for templates that defines a `rule_id`, although technically there is no rule.
Therefore, one can use the rule_id as oval_id right away, with one exception - the sysctl rules are somehow convoluted, and the substitution generates conflicting IDs.